cmd/create.go

```diff
@@ -120,27 +128,29 @@ A full list of component types that can be deployed is available using: 'odo com
 				os.Exit(1)
 			}
 
+			fmt.Printf("Building component %s from binary %s\n\n", componentName, path)
```

Comment:
This message is different than the similar one for local component (line 112).
We should align it together with the message in line 144.

Reply:
Bit confused here... Lines 112 and 144 are the same / similar?

fmt.Printf("Building component %s from directory %s\n", componentName, dir)
fmt.Printf("Building component %s from binary %s\n\n", componentName, path)

Reply:
message on line`112` for the local component doesn't have word "component" in it.
But the others (for binary and local) have. I think they should all have the same wording.